site_scons/prereq_tools/base.py

```diff
@@ -783,12 +792,14 @@ def _setup_intelc(self):
         self.__env.Replace(INTEL_C_COMPILER_VERSION=version)
         self.__env.Replace(LINK=env.get("LINK"))
         # disable the warning about Cilk since we don't use it
-        self.__env.AppendUnique(LINKFLAGS=["-static-intel",
-                                           "-diag-disable=10237"])
-        self.__env.AppendUnique(CCFLAGS=["-diag-disable:2282",
-                                         "-diag-disable:188",
-                                         "-diag-disable:2405",
-                                         "-diag-disable:1338"])
+        if not self.has_icx:
+            self.__env.AppendUnique(LINKFLAGS=["-static-intel",
+                                               "-diag-disable=10237"])
+            self.__env.AppendUnique(CCFLAGS=["-diag-disable:2282",
+                                             "-diag-disable:188",
+                                             "-diag-disable:2405",
+                                             "-diag-disable:1338"])
```

Comment:
This is all legacy stuff now, it's not used any more?

Reply:
Right, I went through their guide on porting.   None of these options are supported in icx